### PR TITLE
Add New Challenge Lab : [ 1740 ] Migrate a MySQL Database to Google Cloud SQL

### PR DESCRIPTION
**Pull request by AguzzTN54**

### CODE REVIEW

File: labs/[1739]_scale_out_and_update_a_containerized_application_on_a_kubernetes_cluster/guide.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Configure Secure RDP using a Windows Bastion Host
-# https://www.qwiklabs.com/focuses/1737?parent=catalog
+# Build and Deploy a Docker Image to a Kubernetes Cluster
```
Comment (elmoallistair) [inline labs/[1739]_scale_out_and_update_a_containerized_application_on_a_kubernetes_cluster/guide.txt:1]: Nice found,  but this must be: 
"Scale Out and Update a Containerized Application on a Kubernetes Cluster"

**Review by elmoallistair — CHANGES_REQUESTED** [review]
Can you fix the Lab name? (1739)

File: labs/[1739]_scale_out_and_update_a_containerized_application_on_a_kubernetes_cluster/guide.txt (hunk 1 above)
Comment (AguzzTN54) [reply labs/[1739]_scale_out_and_update_a_containerized_application_on_a_kubernetes_cluster/guide.txt]: I'm Sorry.
It's already fixed now :)

**Review by elmoallistair — APPROVED** [review]

**Comment by elmoallistair** [comment]
All nice, thanks

**Pull request merged**